javascript/javascript1/week3/SmartPhoneUsageApplication.js

```diff
@@ -0,0 +1,56 @@
+//Adding an activity
+const activities = [];
+
+function addActivity(date , activity , duration)
+{
+    activities.push(
+        {
+            date :date , 
+            activity : activity , 
+            duration : duration
+        }
+    )
+    
+    console.log(activities);
+}
+addActivity("23/7-18", "Youtube", 30);
+addActivity("22/7-18", "Whatsapp", 20);
+addActivity("21/7-18", "Facebook", 28);
+addActivity("21/7-18", "Instagram", 48);
+addActivity("21/7-18", "TikTok", 38);
+
+//Show my status
+function showStatus(activities)
+{ 
+    let totalDuration = 0;
+    if(activities.length !== 0)
+    {      
+        for(i = 0 ; i < activities.length ; i++ )
```

Comment:
You don't need this if-statement.
This for-loop will only run as long as `i` is lower than the length of the array, and since 0 isn't lower than 0, the loop doesn't run.

Reply:
Modified